packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import flexStyles from '@patternfly/patternfly-next/utilities/Flex/flex.css';
+import displayStyles from '@patternfly/patternfly-next/utilities/Display/display.css';
+import alignmentStyles from '@patternfly/patternfly-next/utilities/Alignment/alignment.css';
+import spacingStyles from '@patternfly/patternfly-next/utilities/Spacing/spacing.css';
```

Comment:
I guess there is a similar concern here that this will increase the size of the component too much. Could do something similar as here:
https://github.com/patternfly/patternfly-react/pull/863/files#diff-4e9f1155160426190b5a2ee57a9c0a5fR32
Or use styled-component when the PR is merged :)